data/reusables/actions/github-token-available-permissions.md

```diff
@@ -19,7 +19,7 @@ permissions:
 
 If you specify the access for any of these scopes, all of those that are not specified are set to `none`.
 
-You can use the following syntax to define read or write access for all of the available scopes:
+You can use the following syntax to define one of `read-all` or `write-all` access for all of the available scopes:
 
 ```yaml
 permissions: read-all|write-all
```

Comment:
I think we can also make the examples workable:

````suggestion
You can use the following syntax to define one of `read-all` or `write-all` access for all of the available scopes:

```yaml
permissions: read-all
```

```yaml
permissions: write-all
````